### PR TITLE
[#1501] Create Unit tests for FeedbackRubricQuestionDetails Class

### PR DESCRIPTION
**Pull request by vitormanoel17**
Fixes Part of #1501 

Adding unit tests for the shouldChangesRequireResponseDeletion method in the FeedbackRubricQuestionDetails Class

### CODE REVIEW

File: src/test/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetailsTest.java [hunk 1]
```diff
@@ -23,6 +23,107 @@ public void testConstructor_defaultConstructor_fieldsShouldHaveCorrectDefaultVal
         assertTrue(rubricDetails.getRubricWeights().isEmpty());
     }
 
+    //V
+    @Test
+    public void testShouldChangesRequireResponseDeletion_emptyRubricChoices_shouldReturnTrue() {
+        FeedbackRubricQuestionDetails rubricDetails = new FeedbackRubricQuestionDetails();
+        rubricDetails.setRubricChoices(Arrays.asList("test-1", "test-2"));
+        FeedbackRubricQuestionDetails newrubricDetails = new FeedbackRubricQuestionDetails();
```
Comment (zhaojj2209) [inline src/test/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetailsTest.java:31]:
```suggestion
        FeedbackRubricQuestionDetails newRubricDetails = new FeedbackRubricQuestionDetails();
```
Do make the same change to the other testcases.

File: src/test/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetailsTest.java [hunk 2]
```diff
@@ -23,6 +23,107 @@ public void testConstructor_defaultConstructor_fieldsShouldHaveCorrectDefaultVal
         assertTrue(rubricDetails.getRubricWeights().isEmpty());
     }
 
+    //V
```
Comment (zhaojj2209) [inline src/test/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetailsTest.java:26]: Please remove the comments on top of each of the new testcases.

**Review by zhaojj2209 — COMMENTED** [review]
Thanks for the additional tests, just some nits.

**Review by zhaojj2209 — APPROVED** [review]
LGTM

**Review by wkurniawan07 — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**